include/swift/Strings.h

```diff
@@ -30,6 +30,8 @@ constexpr static const StringLiteral SWIFT_DISTRIBUTED_NAME = "Distributed";
 constexpr static const StringLiteral SWIFT_STRING_PROCESSING_NAME = "_StringProcessing";
 /// The name of the SwiftShims module, which contains private stdlib decls.
 constexpr static const StringLiteral SWIFT_SHIMS_NAME = "SwiftShims";
+/// The name of the CxxShim module, which contains cxx stdlib imports.
```

Comment:
It actually contains a casting utility (see `swift/stdlib/public/Cxx/cxxshim/libcxxshim.h`), can you please update the comment?